kernel/comps/virtio/src/device/network/device.rs

```diff
@@ -170,29 +191,68 @@ fn queue_to_network_error(err: QueueError) -> VirtioNetError {
     match err {
         QueueError::NotReady => VirtioNetError::NotReady,
         QueueError::WrongToken => VirtioNetError::WrongToken,
+        QueueError::BufferTooSmall => VirtioNetError::Busy,
         _ => VirtioNetError::Unknown,
     }
 }
 
+fn init_caps(features: &NetworkFeatures, config: &VirtioNetConfig) -> DeviceCapabilities {
+    let mut caps = DeviceCapabilities::default();
+
+    caps.max_burst_size = None;
+    caps.medium = Medium::Ethernet;
+
+    if features.contains(NetworkFeatures::VIRTIO_NET_F_MTU) {
+        // If `VIRTIO_NET_F_MTU` is negotiated, the MTU is decided by the device.
+        caps.max_transmission_unit = config.mtu as usize;
+    } else {
+        // Linux does not has a specified MTU for virtio-net.
+        // Linux only requires the MTU between 68-65535.
+        // The exact value can be decided dynamically.
+        // <https://elixir.bootlin.com/linux/v6.0.9/source/drivers/net/virtio_net.c#L3660>.
```

Comment:
The comments also seem useless and misleading because the MTU values do not come from the Linux kernel. 

When I saw the comments, I tried to find the values in the Linux kernel, but failed.